src/util/math.ts

```diff
@@ -15,6 +15,7 @@ export function accuratePow64(x: f64, y: f64): f64 {
     // or/and 10 ** -5 != 1e-5 anymore. For avoid this behaviour we are forcing exponent
     // to fractional form and compensate this afterwards.
     if (isFinite(y) && Math.abs(y) >= 2 && Math.trunc(y) == y) {
+      if (x < 0) return -accuratePow64(-x, y);
```

Comment:
Something like this should fix `accuratePow64(-3, 2)`

```suggestion
      if (x < 0) return y % 2 ? -accuratePow64(-x, y) : accuratePow64(-x, y);
```